paper/paper.md

```diff
@@ -124,44 +125,148 @@ The table presents the suggested visualisation methods (geoms) based on the data
 In order to produce a `{funkyheatmap}` visualisation, you need to provide the data in the form of a dataframe, which also must contain a column named `id`.
 If you provide no other information, a basic visualisation will be provided, but customization is possible by provinding additional information, such as a `column_info` dataframe which details how the columns in the dataframe get translated into different geoms, or a `row_groups` dataframe which allows you to group rows in the visualisation.
 
-As an example, the following code produces \autoref{fig-mtcars}.
+As an example, the following code produces \autoref{fig-mtcars}. For an in-depth explanation of how we come to 
+all these data structures and tribbles, please refer to the vignette named `mtcars`.
```

Comment:
What about this? I feel like the full source code is a bit too much for the scope of the JOSS manuscript -- I'd prefer to keep it simple. WDYT?

```suggestion
As an example, the following code produces \autoref{fig-mtcars}. For an in-depth explanation on how to fine-tune this figure and the different data structures involved, please see [`vignette("mtcars", "funkyheatmap")`](https://funkyheatmap.github.io/funkyheatmap/articles/mtcars.html).
```

Reply:
That would be fine by me. This is not the full source code by the way, these are the data structures you end up with if you follow the vignette, but not the steps on how to get there. But it is a bit long indeed.